From 2268e0f0f372a61827b912155a1796490968b3ff Mon Sep 17 00:00:00 2001
From: SuperBFG7 <daniel@despite.ch>
Date: Tue, 2 Jun 2020 12:41:52 +0200
Subject: [PATCH] added forward declarations


[Retrieved (and slightly updated to remove update of dirble_api_token)
from:
https://github.com/notandy/ympd/pull/191/commits/2268e0f0f372a61827b912155a1796490968b3ff]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/mpd_client.c | 2 ++
 src/mpd_client.h | 6 ++++--
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/mpd_client.c b/src/mpd_client.c
index 2911e461..9f003ab0 100644
--- a/src/mpd_client.c
+++ b/src/mpd_client.c
@@ -30,6 +30,8 @@
 
 /* forward declaration */
 static int mpd_notify_callback(struct mg_connection *c, enum mg_event ev);
+char dirble_api_token[28];
+struct t_mpd mpd;
 
 const char * mpd_cmd_strs[] = {
     MPD_CMDS(GEN_STR)
diff --git a/src/mpd_client.h b/src/mpd_client.h
index 447dd563..9342c550 100644
--- a/src/mpd_client.h
+++ b/src/mpd_client.h
@@ -96,6 +96,8 @@ struct t_mpd {
 
     int song_id;
     unsigned queue_version;
-} mpd;
+};
+
+extern struct t_mpd mpd;
 
 struct t_mpd_client_session {
     int song_id;
